scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -397,11 +397,11 @@ download_cncli() {
   [[ -z ${ARCH##*aarch64*} ]] && err_exit "  The cncli pre-compiled binary is not available for ARM, you might need to build them!"
   echo -e "\nInstalling CNCLI.."
   if command -v cncli >/dev/null; then cncli_version="v$(cncli -V 2>/dev/null | cut -d' ' -f2)"; else cncli_version="v0.0.0"; fi
-  cncli_git_version="$(curl -s https://api.github.com/repos/${G_ACCOUNT}/cncli/releases/latest | jq -r '.tag_name')"
+  cncli_git_version="$(curl -s https://api.github.com/repos/cardano-community/cncli/releases/latest | jq -r '.tag_name')"
```

Comment:
I believe this is to avoid a fork of cncli being required when building the container in an SPO's forked repo?

Reply:
correct, i did not have the cncli build and retrievable from my fork, so i took it from cardano-community, can/should this be build and deployed in my fork? is there documents on this process? would be happy comply and revert this.

Reply:
maybe I needed to fork the cncli repo, in addition, to resolve this properly.

Reply:
ref: [986a091ad38ce455c14c1ff3bf253f402e15d5c2](https://github.com/cardano-community/guild-operators/commit/986a091ad38ce455c14c1ff3bf253f402e15d5c2)

Reply:
@rdlrt  appears to be who added the commit using G_ACCOUNT. I suspect the intent was to allow forked testing of cncli changes. Most of @AndrewWestberg branches appear to be local to cardano-community/cncli, not from a a fork. Howvever, I am not sure that's a good indicator this change won't impact other contributors if its hard coded to cardano-community.

I'll defer to @rdlrt @AndrewWestberg and @Scitz0 on this for further clarification.

Reply:
I reverted my change here. in general i think this is good practice to have G_ACCOUNT in place for all 'cardano-community' instances if Forked testing is expected. G_ACCOUNT=cardano-community can always be overridden.

Reply:
Makes sense to leave it as hard-coded cardano-community for cncli - since the changes to that repo will likely come against fork there